test/hotspot/jtreg/vmTestbase/nsk/monitoring/MemoryPoolMBean/isCollectionUsageThresholdExceeded/isexceeded001.java

```diff
@@ -120,15 +120,11 @@ public static int run(String[] argv, PrintStream out) {
                 isExceeded = monitor.isCollectionThresholdExceeded(pool);
                 if (!isExceeded) {
 
-                    // Refresh the values
-                    threshold = monitor.getCollectionThreshold(pool);
-                    usage = monitor.getCollectionUsage(pool);
-                    if (used >= threshold) {
```

Comment:
Although `used` is not updated, `threshold` is. Couldn't removing this extra check result in new failures?

Reply:
I think that's OK.  We set the threshold earlier in the test, to the value of used+1, we don't need to read threshold back again.  I see in testing that it reads back the value that it set, as it should, this value isn't changed again.  

(The test's monitor.setCollectionThreshold method (MemoryMonitor.java) for setting this ends up calling into MemoryPoolImpl.java)

The testing at lines 116 - 121 is checking if we have crossed the threshold, and if so then isExceeded really should be true.  It's surviving testing so far!